finality-aleph/src/network/gossip/service.rs

```diff
@@ -41,7 +39,7 @@ enum Command<D: Data, P: Clone + Debug + Eq + Hash + Send + 'static> {
 ///   2. Various forms of (dis)connecting, keeping track of all currently connected nodes.
 /// 3. Outgoing messages, sending them out, using 1.2. to broadcast.
 pub struct Service<N: RawNetwork, ES: EventStream<N::PeerId>, AD: Data, BSD: Data> {
-    network: N,
+    _network: N,
```

Comment:
If you don't need it then uh, get rid of it? or do you need to keep the object for some reason?

Reply:
done